tests/io/FileFactoryTest.py

```diff
@@ -42,15 +42,12 @@ def test_deprecated_dict_getitem(self):
 
     def test_lines(self):
         self.assertEqual(self.uut.lines, ('This is a test file.\n',))
-        self.assertEqual(self.uut.lines, ('This is a test file.\n',))
 
```

Comment:
I still don't remember why I added duplicate asserts maybe I thought that it would check whether the properties were cached or not :sweat_smile: , but now that I look at it they seem redundant so I'm okay with deleting these. :+1:

Reply:
Yeah, seems they were added in this commit https://github.com/coala/coala/commit/fc568a01ba489376b4d6efac12ea772cb3cb1573#diff-a4fb13da1735371668368c3a091d01e2